boards/arm/stm32/stm32f4discovery/configs/adb/defconfig

```diff
@@ -44,21 +44,25 @@ CONFIG_INTELHEX_BINARY=y
 CONFIG_LIBC_ENVPATH=y
 CONFIG_LIBC_EXECFUNCS=y
 CONFIG_LIBUV=y
-CONFIG_LIBUV_PIPE=y
-CONFIG_LIBUV_STREAM=y
 CONFIG_MMCSD=y
+CONFIG_NET=y
+CONFIG_NETDEV_LATEINIT=y
+CONFIG_NET_SOCKOPTS=y
+CONFIG_NET_TCP_NO_STACK=y
+CONFIG_NET_UDP_NO_STACK=y
 CONFIG_NSH_ARCHINIT=y
 CONFIG_NSH_BUILTIN_APPS=y
 CONFIG_NSH_FILEIOSIZE=512
 CONFIG_NSH_FILE_APPS=y
 CONFIG_NSH_LINELEN=128
 CONFIG_NSH_READLINE=y
 CONFIG_PATH_INITIAL="/mnt/sd0/bin"
-CONFIG_PIPES=y
 CONFIG_PREALLOC_TIMERS=4
+CONFIG_PSEUDOFS_SOFTLINKS=y
 CONFIG_RAM_SIZE=114688
 CONFIG_RAM_START=0x20000000
 CONFIG_RAW_BINARY=y
+CONFIG_SCHED_HAVE_PARENT=y
```

Comment:
why need

Reply:
Removed, some boards need --gc-sections to remove dead code that caused link error.